pycontrails/datalib/gfs/variables.py

```diff
@@ -43,7 +43,7 @@
 
 
 TOAUpwardShortwaveRadiation = MetVariable(
-    short_name="uswrf",
+    short_name="suswrf",
```

Comment:
This is not correct -  I'm not sure what variable `suswrf` is, but it may be "surface upward shortwave radiative flux"?

I grabbed a random forecast file from [nomad](https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gfs.20241014/06/atmos/) and the current implementation still looks correct.

See [GFS docs](https://www.nco.ncep.noaa.gov/pmb/products/gfs/gfs.t00z.pgrb2.0p25.f003.shtml) (Search for `USWRF`, etc)

I ran the following and it worked fine:

```
from pycontrails.models.cocip import Cocip
from pycontrails.datalib.gfs import GFSForecast

time_bounds = ("2024-10-14 00:00:00", "2024-10-14 01:00:00")

gfs_rad = GFSForecast(time_bounds, variables=Cocip.rad_variables, show_progress=True)
rad = gfs_rad.open_metdataset()
```

Reply:
Do you have a source for the variable name change? Or did you find it by inspection? And if so, where do you find it?

Reply:
I think I've figured out what's going on. The issue isn't a change in the grib files themselves; it's a change in [eccodes](https://confluence.ecmwf.int/display/MTG2US/Changes+in+ecCodes+version+2.38.0+compared+to+the+previous+version#ChangesinecCodesversion2.38.0comparedtothepreviousversion-Changedshortnames), which I only just found. I'll include more in this PR saying as much and requiring the latest eccodes.

I think if you were to upgrade eccodes and rerun your snippet with a different time (or delete the cached files), I expect you to see a failure.

Reply:
Wow thats incredibly annoying! Its a little confusing because `suswrf` *does* mean "Surface upward short-wave radiation flux", but because of the way we are using the GFS variable, its the "nominal top" one instead of the "surface one."

Probably worth putting a comment in there, I'll make a suggestion